tests/fast-integration/skr-svcat-migration-test/fixtures/02_serviceinstance_auditlog.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: svcat-auditlog-api-1
+  namespace: default
```

Comment:
Remove namespace from yaml. You won't be able to install it in another namespace with current implementation of kubectlApply (namespace is applied only for resources where the namespace is not given).